src/config/Config.h

```diff
@@ -98,4 +98,4 @@ class Config {
     bool mIsNewInstance;
     std::map<uint32_t, std::shared_ptr<ConfigVersionUpdater>> mVersionUpdaters;
 };
-} // namespace LUS
+} // namespace LUS
```

Comment:
not having a newline at the end of this file is making it show up as part of the diff when it hasn't been changed. it'll be nice to clean this up for a clean diff

```suggestion
} // namespace LUS

```